src/bbs2gh/Commands/MigrateRepoCommand.cs

```diff
@@ -14,27 +14,21 @@ public class MigrateRepoCommand : Command
         public MigrateRepoCommand(
             OctoLogger log,
             GithubApiFactory githubApiFactory,
-            EnvironmentVariableProvider environmentVariableProvider) : base("migrate-repo")
+            EnvironmentVariableProvider environmentVariableProvider
+        ) : base("migrate-repo")
         {
             _log = log;
             _githubApiFactory = githubApiFactory;
             _environmentVariableProvider = environmentVariableProvider;
 
-            Description = "Invokes the GitHub API's to migrate the repo and all PR data";
+            Description = "Import a Bitbucket Server archive to GitHub.";
             Description += Environment.NewLine;
-            Description += "Note: Expects ADO_PAT and GH_PAT env variables or --ado-pat and --github-pat options to be set.";
+            Description += "Note: Expects GH_PAT env variable or --github-pat option to be set.";
 
-            var adoOrg = new Option<string>("--ado-org")
+            var archiveUrl = new Option<string>("--archive-url")
             {
-                IsRequired = true
-            };
-            var adoTeamProject = new Option<string>("--ado-team-project")
-            {
-                IsRequired = true
-            };
-            var adoRepo = new Option<string>("--ado-repo")
-            {
-                IsRequired = true
+                IsRequired = true,
+                Description = "URL used to downlodad Bitbucket Server migration archive."
```

Comment:
typo in download

Reply:
Good catch! Fixed in 52f63c12bb33b397cdc567dd3854a7fb2e3ad0bf!